test/src/loaders/gtdb/gtdb_genome_stats_loader_test.py

```diff
@@ -0,0 +1,72 @@
+import inspect
```

Comment:
I'm noticing some common functions between this test file and the one for the taxa freq loader. Maybe next time you deal with tests extract some helper fns into common methods?

Reply:
👍 I will work on some restructuring work.

Reply:
:+1: